.github/workflows/publish-nightly.yml

```
@@ -185,13 +185,28 @@ jobs:
+            echo "available=true" >> $GITHUB_OUTPUT
+          else
+            echo "available=false" >> $GITHUB_OUTPUT
+          fi
+
       - name: Checkout Workflow Repository
         uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6
         with:
           fetch-depth: 1
 
       - name: Build Full Changelog
+        if: ${{ steps.webhook_secrets_available.outputs.available == 'true' }}
```

actionlint (1.7.12)

[error] property "webhook_secrets_available" is not defined in object type {webhook_secrets_check: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)